src/Curry.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types=1);
```

Comment:
But as per the note, it only makes a difference if set in the file calling the methods.

E.g.
```PHP
// foo.php
<?php
declare(strict_types=1);

class A
{
    public function test(int $a): int
    {
        return $a;
    }

    public function test2(int $a): int
    {
        return $this->test($a);
    }
}
```
```PHP
// bar.php
<?php

require_once __DIR__ . '/foo.php';

$a = new A();
echo $a->test2('1'); // (int) 1
```

if you take the `declare(strict_types=1);` out of the `foo.php` file it still returns `(int) 1`